src/validation/multicityValidation.js

```diff
@@ -0,0 +1,31 @@
+import { body, validationResult } from 'express-validator';
+
+const multicityCheck = [
+  body('departureDate').trim().not().isEmpty()
+    .withMessage('Departure Date field is required')
+    .matches(/^\d{4}([-./,:])\d{2}\1\d{2}$/, 'i')
+    .withMessage('The date must follow date format YYYY-MM-DD'),
+  body('returnDate').trim().not().isEmpty()
+    .withMessage('Return Date field is required')
+    .matches(/^\d{4}([-./,:])\d{2}\1\d{2}$/, 'i')
+    .withMessage('The date must follow date format YYYY-MM-DD'),
+  body('currentOfficeLocation').trim().not().isEmpty()
+    .withMessage('current Office Location field is required'),
+  body('reason').trim().not().isEmpty()
+    .withMessage('Reason field is required'),
+  body('tripType').trim().not().isEmpty()
+    .withMessage('The trip type field is required')
```

Comment:
I noticed the triptype is called `Multiple stops`. We should try to stick to one word here like `multicity`. Also, you should test that the value passed as tripType is `multicity`

Reply:
I implemented this feedback with @Stylll